simple_config

```diff
@@ -12,5 +12,10 @@ path /echo EchoRequestHandler {}
 
 path /status StatusHandler {}
 
+path /proxy ProxyRequestHandler {
+  proxy_host  ucla.edu;
```

Comment:
And finally, just for consistency, we can call this `reverse_proxy_host`, which would prevent us accidentally mixing up host/port settings between proxy and reverse-proxy config-blocks, for example.